src/sentry/tasks/integrations/github/open_pr_comment.py

```diff
@@ -293,7 +315,9 @@ def get_top_5_issues_by_count_for_file(
                     Column("group_id"),
                     Function("count", [], "event_count"),
                     Function(
-                        "arrayElement", (Column("exception_frames.function"), -1), "function_name"
+                        "multiIf",
```

Comment:
`multi_if`?

Reply:
this is how you format `multiIf` in `SnubaRequests`